src/cai_lab_to_nwb/zaki_2024/zaki_2024_metadata.yaml

```diff
@@ -37,5 +37,22 @@ NWBFile:
     - Zaki, Yosif
 Subject:
   species: Mus musculus
-  age: TBD  # in ISO 8601, such as "P1W2D"
-  sex: TBD  # One of M, F, U, or O
+  age: P12W/P15W  # in ISO 8601, such as "P1W2D"
```

Comment:
Is this from the paper?

Reply:
Yes, but I will confirm it in the upcoming meeting. I need it defined to run the conversion